src/FlaUI.Core.UITests/Elements/MenuTests.cs

```diff
@@ -29,9 +29,20 @@ public void TestMenuWithSubMenus()
             Assert.That(subitems1, Has.Length.EqualTo(1));
             Assert.That(subitems1[0].Properties.Name, Is.EqualTo("Exit"));
             var subitems2 = items[1].Items;
-            Assert.That(subitems2, Has.Length.EqualTo(2));
+            if (ApplicationType == TestApplicationType.WinForms)
+            {
+                Assert.That(subitems2, Has.Length.EqualTo(2));

```

Comment:
Can you maybe add some comments why this is different?

Reply:
Done.